lib/scanner-node.js

```diff
@@ -0,0 +1,267 @@
+const _ = require('lodash/fp')
+
+const Pdf417Parser = require('./compliance/parsepdf417')
+const { utils: coinUtils } = require('@lamassu/coins')
+const { cameraExists, stream } = require('./capture/streamer/v4l2camera')
+const scanner = require('./capture/scanner/manatee')
+const sharp = require('sharp')
+
+const DEFAULT_FPS = 10
+const DEFAULT_DELAYEDSHOT_DELAY = 3
+
+let configuration = null
+let current_fps = DEFAULT_FPS
+let delayedshot_delay = DEFAULT_DELAYEDSHOT_DELAY
+
+let activeStream = null
+
+const maxCamResolutions = [
```

Comment:
`maxCamResolutions`, `maxCamResolutions`, and `maxCamResolutionQRCode` are defined in `consts.js`. Maybe `maxCamResolutionPhotoId` could be added there too, though it might currently not be used anywhere else.